src/BuiltinExtensions/ImageBatchTool/ImageBatchToolExtension.cs

```diff
@@ -115,6 +115,12 @@ void removeDoneTasks()
         {
             string fname = file.Replace('\\', '/').AfterLast('/');
             int imageIndex = batchId++;
+            // the following two variables are used to handle when one input file
```

Comment:
these `//` comments are still really not needed

Reply:
will yank